---
layout: post
title: RECOMB Awards
permalink: /awards/
isStaticPost: true
---

 
#### Best Paper Awards

*   2025: Enzo Battistella, Anant Maheshwari, Barış Ekim, Bonnie Berger and Victoria Popic. ["ralphi: a deep reinforcement learning framework for haplotype assembly"](https://link.springer.com/chapter/10.1007/978-3-031-90252-9_37).
*   2024: Li Song and Ben Langmead. ["Centrifuger: lossless compression of microbial genomes for efficient and accurate metagenomic sequence classification"](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03244-4).
*   2023: Sara C. Schulte, Alexander Dilthey and Gunnar W. Klau. ["HOGVAX: Exploiting peptide overlaps to maximize population coverage in vaccine design with application to SARS-CoV-2"](https://www.biorxiv.org/content/10.1101/2023.01.09.523288v1).
*   2021: Serhan Yılmaz, Marzieh Ayati, Daniela Schlatzer, A. Ercüment Çiçek, Mark R. Chance and Mehmet Koyutürk. ["Robust inference of kinase activity using functional networks"](https://www.nature.com/articles/s41467-021-21211-6).
*   2020: Amatur Rahman and Paul Medvedev. ["Representation of k-mer sets using spectrum-preserving string sets"](https://www.liebertpub.com/doi/10.1089/cmb.2020.043).
*   2019: Martin J. Zhang, Fei Xia, James Zou. ["AdaFDR: a fast, powerful and covariate-adaptive approach to multiple hypothesis testing"](https://www.nature.com/articles/s41467-019-11247-0).
*   2018: Sebastien Roch and Kun-Chieh Wang. ["Circular networks from distorted metrics"](https://link.springer.com/chapter/10.1007/978-3-319-89929-9_11).
*   2017: Yijie Wang, Dong-Yeon Cho, Hangnoh Lee, Brian Oliver and Teresa Przytycka. ["NetREX: Network Rewriting using EXpression - towards context specific regulatory networks"](https://www.nature.com/articles/s41467-018-06382-z).
*   2016: Emre Sefer and Ziv Bar-Joseph. ["Shall we dense? Comparing design strategies for time series expression experiments"](https://www.sciencedirect.com/science/article/pii/S2405471216302125).
*   2015: David Manescu and Uri Keich. ["A symmetric length-aware enrichment test"](https://link.springer.com/chapter/10.1007/978-3-319-16706-0_23).
*   2014: Jianzhu Ma, Sheng Wang, Zhiyong Wang and Jinbo Xu. ["MRFalign: protein homology detection through alignment of markov random fields"](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003500).
*   2013: Fabio Vandin, Alexandra Papoutsaki, Ben Raphael, and Eli Upfahl. ["Genome-wide survival analysis of somatic mutations in cancer"](https://link.springer.com/chapter/10.1007/978-3-642-37195-0_26).
*   2012: Patrick Holloway, Krister Swenson, David Ardell and Nadia El-Mabrouk for the contribution ["Evolution of genome organization by duplication and loss: a linear programming approach"](https://link.springer.com/chapter/10.1007/978-3-642-29627-7_10).
*   2011: Shay Zakov, Yoav Goldberg, Michael Elhadad and Michal Ziv-Ukelson. ["Rich parametrization improves RNA structure prediction"](https://www.liebertpub.com/doi/10.1089/cmb.2011.0184).
*   2010: Nir Atias and Roded Sharan. ["An algorithmic framework for predicting side-effects of drugs"](https://link.springer.com/chapter/10.1007/978-3-642-12683-3_1).

#### Best Student / Young Scientist Paper Awards

*   2025: Kemal Inecik, Aleyna Kara, Antony Rose, Muzlifah Haniffa and Fabian Theis. ["TarDis: Achieving Robust and Structured Disentanglement of Multiple Covariates"](https://link.springer.com/chapter/10.1007/978-3-031-90252-9_23).
*   2024: Matthew M. Hong, David Froelicher, Ricky Magner, Victoria Popic, Bonnie Berger, and Hyunghoon Cho. ["Secure Discovery of Genetic Relatives across Large-Scale and Distributed Genomic Datasets"](https://doi.org/10.1101/gr.279057.124).
*   2023: Sara C. Schulte, Alexander Dilthey and Gunnar W. Klau. ["HOGVAX: Exploiting peptide overlaps to maximize population coverage in vaccine design with application to SARS-CoV-2"](https://www.biorxiv.org/content/10.1101/2023.01.09.523288v1).
*   2021: Barış Ekim, Bonnie Berger and Rayan Chikhi. ["Minimizer-space de Bruijn graphs: whole-genome assembly of long reads in minutes on a personal computer"](https://www.sciencedirect.com/science/article/pii/S240547122100332X).
*   2018: Ariya Shajii, Ibrahim Numanagić and Bonnie Berger. ["Latent variable model for aligning barcoded short-reads improves downstream analyses"](https://www.sciencedirect.com/science/article/pii/S2405471218302849).
*   2014: Emily Berger, Deniz Yorukoglu, Jian Peng and Bonnie Berger. [“HapTree: A Novel Bayesian Framework for Single Individual Polyplotyping using NGS Data”](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003502).
*   2004: Eran Segal and Roded Sharan. ["A discriminative model for identifying spatial cis-regulatory modules"](https://dl.acm.org/doi/10.1145/974614.974634).
*   2003: Gill Bejerano. ["Efficient exact value computation and applications to biosequence analysis"](https://dl.acm.org/doi/10.1145/640075.640080).
*   2001: Mathieu Blanchette. ["Algorithms for phylogenetic fingerprinting"](https://dl.acm.org/doi/10.1145/369133.369170).

#### Best Poster Awards

*   2025: Nayoung Park, Younhee Ko, Jaebum Kim. "MetaCNVNet: A hybrid deep learning framework with multi-caller integration for enhanced copy number variation detection".
*   2024:
    *   Jim Shaw and Yun William Yu. "sylph: metagenome profiling by statistical k-mer containment".
    *   Aidan Lakshman and Erik Wright. "EvoWeaver: Large-scale prediction of gene functional associations from coevolutionary signals".
*   2023: Alex Sweeten, Mike Schatz, Adam Phillippy. "Mod.Plot: A Rapid and Interactive Visualization of Tandem Repeats".
*   2022: Daniel Ariad, Svetlana Madjunkova and Rajiv McCoy. "Inference of meiotic recombination events from a large sample of human embryos".
*   2021: Arun Das and Michael Schatz. "Analysis of sketching and sampling approaches for fast and accurate long read classification".
*   2020: Emilio Dorigatti and Benjamin Schubert. "Joint epitope selection and spacer design for strings-of-beads vaccine".
*   2019: Chirag Jain, Sanchit Misra, Haowen Zhang, Alexander Dilthey and Srinivas Aluru. "Accelerating sequence alignment to graphs".
*   2018: Luca Denti, Raffaella Rizzi, Stefano Beretta, Gianluca Della Vedova, Marco Previtali and Paola Bonizzoni. "ASGAL: aligning RNA-Seq Data to a splicing graph to detect novel alternative splicing events".
*   2017: Kavya Vaddadi, Naveen Sivadasan, Kshitij Tayal and Rajgopal Srinivasan. "Sequence alignment on directed graphs".
*   2016: Ilia Minkin, Son Pham and Paul Medvedev. "TwoPaCo: an efficient algorithm to build the compacted de Bruijn graph from many complete genomes".
*   2015: Vladimir Shchur and Richard Durbin. "Tree consistent PBWT and their application to reconstructing ancestral recombination graphs and demographic inference".
*   2014:
    *   Patrick Flaherty and Yuting He. "RVD2: An ultra-sensitive variant detection model for low-depth targeted next-generation sequencing data".
    *   Marinka Zitnik and Blaz Zupan. "Biomedical data fusion by simultaneous matrix tri-factorization".
*   2013:
    *   Rongxin Fang and Zhihua Zhang. "Functional distinctive CTCF bindings revealed by a novel motif discovery pipeline".
    *   Tomislav Ilicic and Richard Durbin. "Implementation of efficient haplotype matching using suffix array based methods".
*   2012:
    *   Matthew Edwards and David Gifford. "High-resolution genetic mapping with pooled sequencing".
    *   Thomas Bonfert, Gergely Csaba, Ralf Zimmer and Caroline C. Friedel. "A context-based approach to identify the most likely mapping for RNA-seq experiments".
*   2011:
    *   Chen Yanover and Philip Bradley. "Large-scale characterization of binding landscapes using structural simulations: application to C2H2 zinc-finger transcription factors and MHC proteins".
    *   Raheleh Salari and Teresa M Przytycka. "The effect of SNPs on Boltzmann distribution for RNA secondary structure".
*   2008: Xiao-Li Li and See-Kiong Ng. "Mining for Domain Dependency Sets from Protein Interactions".
*   2004: 
    *   Matthew W. Dimmic, Melissa J. Todd, Carlos D. Bustamante and Rasmus Nielsen. "Detecting correlated amino acid substitutions using Bayesian phylogenetic techniques".
    *   Boris E. Shakhnovich, Eric Deeds, Charles Delisi, and Eugene Shakhnovich. "Protein structure and evolutionary history determine sequence space topology".
*   2003: Andres Figueroa, James Borneman and Tao Jiang. "Clustering binary fingerprint vectors with missing values for DNA array data analysis".


#### Test of Time Awards

*   2025: Sergey Nurk, Anton Bankevich, Dmitry Antipov, Alexey Gurevich, Anton Korobeynikov, Alla Lapidus, Andrey Prjibelsky, Alexey Pyshkin, Alexander Sirotkin, Yakov Sirotkin, Ramunas Stepanauskas, Jeffrey McLean, Roger Lasken, Scott R. Clingenpeel, Tanja Woyke, Glenn Tesler, Max A. Alekseyev, Pavel A. Pevzner. ["Assembling Genomes and Mini-metagenomes from Highly Chimeric Reads"](https://link.springer.com/chapter/10.1007/978-3-642-37195-0_13)
*   2024: Melissa Gymrek, David Golan, Saharon Rosset, and Yaniv Erlich. ["lobSTR: A Short Tandem Repeat Profiler for Personal Genomes"](https://doi.org/10.1101/gr.135780.111). _Presented in RECOMB 2012_
*   2023: Fabio Vandin, Eli Upfal, Ben Raphael. ["De Novo Discovery of Mutated Driver Pathways in Cancer"](https://link.springer.com/chapter/10.1007/978-3-642-20036-6_44). _Presented in RECOMB 2011_
*   2022: Yu Peng, Henry C. M. Leung, S. M. Yiu and Francis Y. L. Chin. ["IDBA – a Practical iterative de Bruijn graph _de novo_ assembler"](https://link.springer.com/chapter/10.1007/978-3-642-12683-3_28). _Published in RECOMB 2010._
*   2021: Nicholas D. Pattengale, Masoud Alipour, Olaf R. P. Bininda-Emonds, Bernard M. E. Moret and Alexandros Stamatakis. ["How many bootstrap replicates are necessary?"](https://link.springer.com/chapter/10.1007/978-3-642-02008-7_13). _Published in RECOMB 2009._
*   2020: Maxim Kalaev, Vineet Bafna and Roded Sharan. ["Fast and accurate alignment of multiple protein networks"](https://link.springer.com/chapter/10.1007/978-3-540-78839-3_21). _Published in RECOMB 2008._
*   2019: Rohit Singh, Jinbo Xu and Bonnie Berger. ["Pairwise global alignment of protein interaction networks by matching neighborhood topology"](https://link.springer.com/chapter/10.1007/978-3-540-71681-5_2). _Published in RECOMB 2007._
*   2018: Steffen Grossmann, Sebastian Bauer, Peter N. Robinson and Martin Vingron. ["An improved statistic for detecting over-represented gene ontology annotations in gene sets"](https://link.springer.com/chapter/10.1007/11732990_9). _Published in RECOMB 2006._
*   2017: Jacob Scott, Trey Ideker, Richard M. Karp, and Roded Sharan. ["Efficient algorithms for detecting signaling pathways in protein interaction networks"](https://link.springer.com/chapter/10.1007/11415770_1). _Published in RECOMB 2005._
*   2016: Roded Sharan, Trey Ideker, Brian P. Kelley and Ron Shamir and Richard M. Karp. ["Identification of protein complexes by comparative analysis of yeast and bacterial protein interaction data"](https://dl.acm.org/doi/10.1145/974614.974652). _Published in RECOMB 2004._
*   2015: Gene Yeo and Christopher B. Burge. ["Maximum entropy modeling of short sequence motifs with applications to RNA splicing signals"](https://dl.acm.org/doi/10.1145/640075.640118). _Published in RECOMB 2003._
*   2014: Amir Ben-Dor, Benny Chor, Richard M. Karp and Zohar Yakhini. ["Discovering local structure in gene expression data: the order-preserving submatrix problem"](https://dl.acm.org/doi/10.1145/565196.565203). _Published in RECOMB 2002._
*   2013:
    *   Jeremy Buhler and Martin Tompa. ["Finding motifs using random projections"](https://dl.acm.org/doi/10.1145/369133.369172). _Published in RECOMB 2001._
    *   Pavel A. Pevzner, Haixu Tang and Michael S. Waterman. ["A new approach to fragment assembly in DNA sequencing"](https://dl.acm.org/doi/10.1145/369133.369230). _Published in RECOMB 2001._
*   2012:
    *   Nir Friedman, Michal Linial, Iftach Nachman and Dana Pe'er. ["Using Bayesian network to analyse expression data"](https://dl.acm.org/doi/10.1145/332306.332355). _Published in RECOMB 2000._
    *   Amir Ben-Dor, Laurakay Bruhn, Nir Friedman,Iftach Nachman, Michèl Schummer and Zohar Yakhini. ["Tissue classification with gene expression profiles"](https://dl.acm.org/doi/10.1145/332306.332328). _Published in RECOMB 2000._
*   2011: Amir Ben-Dor, Ron Shamir and Zohar Yakhini. ["Clustering gene expression patterns"](https://dl.acm.org/doi/10.1145/369133.369167). _Published in RECOMB 2001._
*   2010:
    *   Bonnie Berger and Tom Leighton. ["Protein folding in the hydrophobic-hydrophilic _(HP)_ is NP-complete"](https://dl.acm.org/doi/10.1145/279069.279080). _Published in RECOMB 1998._
    *   Pierluigi Crescenzi, Deborah Goldman, Christos Papadimitriou, Antonio Piccolboni, Mihalis Yannakakis. ["On the complexity of protein folding"](https://dl.acm.org/doi/10.1145/279069.279089). _Published in RECOMB 1998._


<img class="img-responsive feature-image" src="{{ site.baseurl }}/img/posts/award.png" style="display:none">
